lib/doorman.ex

```diff
@@ -1,2 +1,48 @@
 defmodule Doorman do
+  @moduledoc """
+  Provides helpers that take advantage of the options configured in your config
```

Comment:
Changed it to `Provides authentication helpers` but I don't think that's specific enough. I'll probably end up rewriting that part.